book_source/developers_guide/Using-Git.Rmd

```diff
@@ -91,97 +62,90 @@ You can setup SSH keys to make it easier to commit cod back to GitHub. This migh
 
 1. Introduce yourself to GIT
 
-        git config --global user.name "FULLNAME"
-        git config --global user.email you@yourdomain.example.com
+        `git config --global user.name "FULLNAME"`
+        `git config --global user.email you@yourdomain.example.com`
 
 2. Fork PEcAn on GitHub. Goto the PEcAn source code and click on the Fork button in the upper right. This will create a copy of PEcAn in your personal space.
 
 3. Clone to your local machine 
 
-        git clone git@github.com:<username>/pecan.git
+        `git clone git@github.com:<username>/pecan.git`
 
 3. Define upstream repository
  
-        cd pecan
-        git remote add upstream git@github.com:PecanProject/pecan.git
-
-
-## During development:
-
-* commit often; 
-* each commit can address 0 or 1 issue; many commits can reference an issue (see [[Link commits to issue|Using-Git#link-commits-to-issues]])
-* ensure that all tests are passing before anything is pushed into master.
-
-### Basic Workflow
```

Comment:
Likewise, is there a reason the Basic Workflow was deleted rather than being updated to devel instead of master? I agree that the updated Advanced Workflow is the preferred path for folks making large or sustained contributions to PEcAn, but **most** contributors only push small changes infrequently (e.g. 36 people have submitted PR, but only ~10 are on the weekly call) and really aren't going to cause much trouble if they don't create branches for each feature. Git is pretty confusing when you start and the Basic workflow is a nice stepping stone.